conf/model/segmentation/kpconv/KPFCNN.yaml

```diff
@@ -0,0 +1,85 @@
+# @package model
+defaults:
+  - /model/segmentation/default
+
+model:
+  conv_type: "PARTIAL_DENSE"
+  backbone:
+    _target_: torch_points3d.applications.kpconv.KPConv
+    config:
+      define_constants:
+        in_grid_size: 0.02
+        in_feat: 64
+        bn_momentum: 0.2
+        max_neighbors: 25
+      down_conv:
+        down_conv_nn:
+          [
+            [[FEAT + 1, in_feat], [in_feat, 2*in_feat]],
+            [[2*in_feat, 2*in_feat], [2*in_feat, 4*in_feat]],
+            [[4*in_feat, 4*in_feat], [4*in_feat, 8*in_feat]],
+            [[8*in_feat, 8*in_feat], [8*in_feat, 16*in_feat]],
+            [[16*in_feat, 16*in_feat], [16*in_feat, 32*in_feat]],
+          ]
+        grid_size:
+          [  
+            [in_grid_size, in_grid_size],
+            [2*in_grid_size, 2*in_grid_size],
+            [4*in_grid_size, 4*in_grid_size],
+            [8*in_grid_size, 8*in_grid_size],
+            [16*in_grid_size, 16*in_grid_size],
+          ]
+        prev_grid_size:
+          [
+            [in_grid_size, in_grid_size],
+            [in_grid_size, 2*in_grid_size],
+            [2*in_grid_size, 4*in_grid_size],
+            [4*in_grid_size, 8*in_grid_size],
+            [8*in_grid_size, 16*in_grid_size],
+          ]
+        block_names:
+          [
+            ["SimpleBlock", "ResnetBBlock"],
+            ["ResnetBBlock", "ResnetBBlock"],
+            ["ResnetBBlock", "ResnetBBlock"],
+            ["ResnetBBlock", "ResnetBBlock"],
+            ["ResnetBBlock", "ResnetBBlock"],
+          ] 
+        has_bottleneck:
+          [
+            [False, True],
+            [True, True],
+            [True, True],
+            [True, True],
+            [True, True],
+          ]
+        deformable:
+          [
+            [False, False],
+            [False, False],
+            [False, False],
+            [False, False],
+            [False, False],
+          ]
+        max_num_neighbors:
+          [[max_neighbors,max_neighbors], [max_neighbors, max_neighbors], [max_neighbors, max_neighbors], [max_neighbors, max_neighbors], [max_neighbors, max_neighbors]]
+        module_name: KPDualBlock
```

Comment:
Eventually we can redo this to use hydra instantiation for blocks and remove some of the model factory reflection code